### ISSUE TITLE
Compare training loss curves for both nerf-pytorch and HashNeRF-pytorch

### ISSUE THREAD

**Issue by geoffreymantel** [cmpt-985-term-project/term-project#8]
Compare the over-all training curves between the two repositories, to see the impact of the new input encoding function on the rate of convergence to optimal solution.
